custom_components/terncy/const.py

```diff
@@ -27,7 +27,8 @@
 PROFILE_YAN_BUTTON = 6
 PROFILE_SMART_DIAL = 7
 PROFILE_COLOR_LIGHT = 8
-
+# PROFILE_9 = 9  # 中央空调控制器
+PROFILE_10 = 10  # 中央空调分机
```

Comment:
Suggest rename to PROFILE_AC_UNIT_MACHINE

Reply:
ok, renamed